.github/workflows/rust.yml

```diff
@@ -349,9 +349,7 @@ jobs:
           export ARROW_TEST_DATA=$(pwd)/testing/data
           export PARQUET_TEST_DATA=$(pwd)/parquet-testing/data
 
-          # 2020-11-15: There is a cargo-tarpaulin regression in 0.17.0
-          # see https://github.com/xd009642/tarpaulin/issues/618
-          cargo install --version 0.16.0 cargo-tarpaulin
+          cargo install --version 0.18.0-alpha3 cargo-tarpaulin
```

Comment:
I don't understand this change. It seems to cause an error in CI, right?: https://github.com/apache/arrow-datafusion/pull/581/checks?check_run_id=2858343109

Maybe we could put the old version of tarpaulin back?

Reply:
It was failing either way so this line diff was my attempt to fix it in vain

Reply:
I see -- in that case it does seem like disabling code coverage until it is fixed is the right thing to do, and we can file an issue to re-enable it once whatever underlying issue is fixed

Reply:
Filed https://github.com/apache/arrow-datafusion/issues/590 to fix code coverage job. Will disable it via this PR